src/Console/Command/CommandHelp.php

```diff
@@ -269,6 +270,21 @@ public static function toString($value)
         return var_export($value, true);
     }
 
+    public static function getDisplayableAllowedValues(FixerOption $option)
+    {
+        if (null !== $allowed = $option->getAllowedValues()) {
```

Comment:
Ah right, I forgot this one. I will assign the variable before the `if`. Just out of curiosity, why do you want to avoid such syntax? IMO it's more convenient and readable.